packages/gestalt/src/DropdownContextProvider.js

```diff
@@ -0,0 +1,20 @@
+// @flow strict
+import { createContext, type Context } from 'react';
```

Comment:
could we use `docs/src/components/createHydra.js` pattern here?

Reply:
Potentially. What would that gain us here?

Reply:
Standardization across Gestalt?

Reply:
Makes sense. I don't think this is specific to this PR (and can be easily refactored in the future), so let's discuss in our next meeting?